### ISSUE TITLE
- Required input is missing: ckpt_name | Undefined ckpt_name

### ISSUE THREAD

**Issue by Nufzy** [comfyanonymous/ComfyUI#1172]
There are models in the checkpoint folder, but it don't work. I updated the AMD GPU driver and still.

![image](https://github.com/comfyanonymous/ComfyUI/assets/105979076/4a5032a2-d998-4836-880a-fd2dcbc18f0b)
![image](https://github.com/comfyanonymous/ComfyUI/assets/105979076/b6ae8834-710c-4256-9585-e816bf2a8965)

**Comment by chris-aeviator** [comment]
facing the same issue installing comfy today. It picked up the sd_xl_refiner_1.0_0.9vae.safetensors but not 

- lora_example.safetensors
- upscaler.pth

**Comment by Nufzy** [comment]
It turns out that it can work with the processor, but it extremely slow in my case. 
![image](https://github.com/comfyanonymous/ComfyUI/assets/105979076/0a6b12f6-dff6-457a-a958-41c931d7f226)

**Comment by Nufzy** [comment]
I guess it's because i'm trying to run it with RX560. 🥲

**Comment by Camaraarthur** [comment]
Facing the same problem!!! Any solutions?  As far as understood from your comments it is linked to the GPU? I am using a nvidia  A100-40C   (CUDA Version: 12.0 )on linux ubuntu!

**Comment by ltdrdata** [comment]
Does it mean that even after pressing the 'refresh' button, it still shows as "undefined"?

**Comment by Nufzy** [comment]
> Does it mean that even after pressing the 'refresh' button, it still shows as "undefined"?

Yes. I think it is because of the GPU. I tried to run it with processor, using the .bat file, which comes with comfyui, and it worked perfectly. I used colab and it worked well until the limit expired.

**Comment by ruyuhi** [comment]
my computer is RTX 4050,  it's not run also, how do I ?

**Comment by JalilOz** [comment]
Here with a RTX 3070, the same issue

**Comment by 2manyDisplays** [comment]
for me the issue was solved by installing git

**Comment by ltdrdata** [comment]
This issue is known to be caused by a specific custom node. Please try disabling all custom nodes and test again.

**Comment by PathosEthosLogos** [comment]
How about adding debug messages in between the steps?

It works for me regularly, but putting it in Docker seems to make it wonky. There doesn't seem to be any guides out there to fix this either.

**Comment by WWWNoobhere** [comment]
same with rtx3060ti

**Comment by diwansingh-star** [comment]
Load Checkpoint has an unsupported value for ckpt_name
